tla-types/src/main/scala/at/forsyte/apalache/tla/typecomp/ScopedBuilder.scala

```diff
@@ -141,4 +149,68 @@ class ScopedBuilder
         )
     )
   }
+
+  /**
+   * [f EXCEPT ![a1] = e1, ![a2] = e2 ... ![an] = en]
+   *
+   * Is equivalent to {{{[[f EXCEPT ![a1] = e1] EXCEPT ![a2] = e2] EXCEPT ... ![an] = en]}}}
+   */
+  def multiPointExcept(
```

Comment:
```suggestion
  def exceptMany(
```

So it is easy to discover in completion via the most important word "except".

Reply:
Well, the thing is, verbosity is sort-of intended to signal that this is not something we want people using as much in the first place. This one is fine, but `multiDepthExcept` should honestly be avoided.